### PR TITLE
Add curve pool tokens

### PR DESCRIPTION
**Pull request by nebolax**

### CODE REVIEW

File: updates/14/updates.sql [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xb9446c4Ef5EBE66268dA6700D26f96273DE3d571", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xb9446c4Ef5EBE66268dA6700D26f96273DE3d571", "C", "Curve.fi Factory Plain Pool: 3EURpool", "3EURpool-f", 1637061245, NULL, "", "", "0xb9446c4Ef5EBE66268dA6700D26f96273DE3d571");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xEd4064f376cB8d68F770FB1Ff088a3d0F3FF5c4d", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xEd4064f376cB8d68F770FB1Ff088a3d0F3FF5c4d", "C", "Curve CRV-ETH ", "crvCRVETH", 1637742833, NULL, "", "", "0xEd4064f376cB8d68F770FB1Ff088a3d0F3FF5c4d");
+*
```
Comment (yabirgb) [inline updates/14/updates.sql:6]: Are all of them curve pools? If so for them to get their prices queried you will need to add 
```suggestion
INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xEd4064f376cB8d68F770FB1Ff088a3d0F3FF5c4d", 18, "curve_pool"); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xEd4064f376cB8d68F770FB1Ff088a3d0F3FF5c4d", "C", "Curve CRV-ETH ", "crvCRVETH", 1637742833, NULL, "", "", "0xEd4064f376cB8d68F770FB1Ff088a3d0F3FF5c4d");
*
```

If you look for the code that queries the price we check the protocol and if is a curve pool we manage the token in a special way

**Review by yabirgb — COMMENTED** [review]
Two important things

File: updates/14/updates.sql (hunk 1 above)
Comment (nebolax) [reply updates/14/updates.sql:6]: Done, thanks

File: updates/14/updates.sql [hunk 2]
```diff
@@ -0,0 +1,80 @@
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xc270b3B858c335B6BA5D5b10e2Da8a09976005ad", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xc270b3B858c335B6BA5D5b10e2Da8a09976005ad", "C", "Curve.fi Factory USD Metapool: Paxos Dollar (USDP)", "pax-usdp3CRV-f", 1634746746, NULL, "", "", "0xc270b3B858c335B6BA5D5b10e2Da8a09976005ad");
+*
```
[the remark this answers is not in the document]
Comment (nebolax) [reply updates/14/updates.sql]: Fixed, thanks

**Review by yabirgb — APPROVED** [review]
Looks fine already. LGTM

**Pull request merged**